composer.json

```diff
@@ -91,6 +91,7 @@
             "Sonata\\DoctrineORMAdminBundle\\Tests\\": "tests/"
         }
     },
+    "minimum-stability": "alpha",
```

Comment:
We can modify or remove this when we release betas, rcs, and finally the sable version.